.prettierignore

```diff
@@ -10,3 +10,4 @@ CHANGELOG.md
 node_modules
 npm-debug.log
 yarn-error.log
+*.yaml
```

Comment:
why is this change here? 🤔

Reply:
I am not sure, preetier added it I assume.

Reply:
it's not adding it automatically

Reply:
I haven't added it, I did ran yalc though to get core changes. Not sure what caused this.

Reply:
let's remove it from this PR then :)